Orange/widgets/visualize/owmosaic.py

```diff
@@ -331,9 +332,27 @@ def __init__(self):
         self.vizrank, self.vizrank_button = MosaicVizRank.add_vizrank(
             box, self, "Find Informative Mosaics", self.set_attr)
 
+        box2 = gui.vBox(self.controlArea, box="Interior Coloring")
+        """
+        dmod = DomainModel
+        common_options = dict(
+            labelWidth=50, orientation=Qt.Horizontal, sendSelectedValue=True,
+            valueType=str)
+        self.color_model = DomainModel(valid_types=dmod.PRIMITIVE)
+        self.cb_attr_color = gui.comboBox(
+            box2, self, "variable_color",
+            callback=self.update_colors,
+            model=self.color_model, **common_options)
+        """
+        self.cb_attr_color = gui.comboBox(
+            box2, self, value="variable_color",
+            orientation=Qt.Horizontal, contentsLength=12,
+            callback=self.update_colors,
+            sendSelectedValue=True, valueType=str)
+
```

Comment:
The changed GUI is confusing.
- If the combo is set to None, the two radio buttons below have no effect.
- If the radio button is set to "Pearson", then the combo has no effect.

1. Replace the None option in the combo with *Pearson residuals*
2. You can then eliminate the radio buttons. 
3. The checkbox should then be active (only if) combo is not Pearson residuals. If you use DomainModel, you will check for this with, simply, `self.variable_color is not None`.